controllers/tests/config/selfnoderemediationconfig_controller_test.go

```diff
@@ -219,6 +259,10 @@ var _ = Describe("SNR Config Test", func() {
 		config.APIVersion = "self-node-remediation.medik8s.io/v1alpha1"
 		config.Name = "config-defaults"
 		config.Namespace = shared.Namespace
+		BeforeEach(func() {
+			//Config and DS aren't created so no need to clean them
+			isSkipCleanup = true
```

Comment:
a) the lines above should be in this BeforeEach() as well
b) this config is not cleaned up?

Reply:
I've looked into that.
>a) the lines above should be in this BeforeEach() as well

The reason the lines (specificlly `config := &selfnoderemediationv1alpha1.SelfNodeRemediationConfig{}`)  were not in the BeforeEach block, is because the `config` in this context was shadowing the the general config var.
However this config was inteded to be a different instance (that is also the reason it wasn't cleaned, clean would have failed because it had a different name)

I've included it in the cleanup by aligning the name to that of the main config, I've also changed the var name to reduce the confusion.

Reply:
I won't block on this, but IMHO instead of shadowing or using a new config, it would be even cleaner and less confusing to move the non default config values of the outer config into BeforeEach() of the "DS installation" context. Then we don't need a BeforeEach at this place at all. WDYT?

Reply:
Sure